queries.sql

```diff
@@ -1,3 +1,51 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+select *
+from animals
+where name like '%mon';
+select *
+from animals
+where date_of_birth between '2016-1-1' and '2019-1-1';
+select *
+from animals
+where neutered = true and escape_attempts < 3;
+select date_of_birth
+from animals
+where name in ('Agumon','Pikachu');
+select date_of_birth, escape_attempts
+from animals
+where weight_kg > 10.5;
+select *
+from animals
+where neutered = true;
+select *
+from animals
+where name <> 'Gabumon';
+select *
+from animals
+where weight_kg between 10.5 and 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT species from animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS null;
+COMMIT;
+SELECT name, species from animals;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+SELECT * from animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT delete_after_2022;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK to delete_after_2022;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
```

Comment:
- [ ] Please kindly include queries to answer the following questions
  - How many animals are there?
  - How many animals have never tried to escape?
  - What is the average weight of animals?
  - Who escapes the most, neutered or not neutered animals?
  - What is the minimum and maximum weight of each type of animal?
  - What is the average number of escape attempts per animal type of those born between 1990 and 2000?